fluid/DeepASR/data_utils/load_data.py

```diff
@@ -0,0 +1,183 @@
+#by zhxfl 2018.01.24
+""" @package docstring
+load speech data from disk
```

Comment:
Please give more detail comments to describe the organization of data and how to load them.